frontend/src/component/feature/FeatureView/FeatureOverview/FeatureOverviewEnvironments/FeatureOverviewEnvironment/EnvironmentHeader/EnvironmentHeader.tsx

```diff
@@ -0,0 +1,76 @@
+import type { FC, ReactNode } from 'react';
+import {
+    AccordionSummary,
+    type AccordionSummaryProps,
+    styled,
+} from '@mui/material';
+import StringTruncator from 'component/common/StringTruncator/StringTruncator';
+import ExpandMore from '@mui/icons-material/ExpandMore';
+
+const StyledAccordionSummary = styled(AccordionSummary)(({ theme }) => ({
+    boxShadow: 'none',
+    padding: theme.spacing(0.5, 3, 0.5, 2),
+    [theme.breakpoints.down(400)]: {
+        padding: theme.spacing(1, 2),
+    },
+}));
+
+const StyledHeader = styled('div')(({ theme }) => ({
+    display: 'flex',
+    width: '100%',
+    color: theme.palette.text.primary,
+    alignItems: 'center',
+    minHeight: theme.spacing(8),
+}));
+
+const StyledHeaderTitle = styled('div')(({ theme }) => ({
```

Comment:
If this is rendered inside a `h2`, then `div` isn't a valid element (but `span` is).

Anyway, this **might** (emphasis on might) be a good use case for [hgroup](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/hgroup).

Reply:
I wrapped accordion 'summary' in a `header` tag, and added `h2` on environment name

Reply:
Yeah, I'll take that!